app/frontend/src/registry/components/people/OrganizationAdd.vue

```diff
@@ -253,6 +257,9 @@ export default {
         website_url: [],
         email: []
       }
+    },
+    formatTel (value) {
```

Comment:
Can we move this into a common area somewhere? It's copied into 4 different places.